planner_cspace/src/planner_3d.cpp

```diff
@@ -1342,6 +1348,7 @@ class Planner3dNode
         pub_path_.publish(path);
       }
       pub_status_.publish(status_);
+      diag_updater_.force_update();
```

Comment:
https://github.com/at-wat/neonavigation/blob/1f1d5888eabc782256f8b122607a1651a2bb4995/planner_cspace/src/planner_3d.cpp#L384-L386 also needs `force_update()`.

Reply:
or, this one can be `update()` to limit diag publish rate.

Reply:
ok, done. 
FYI, The `update()` limits publish rate less than 1.0 sec. by default (we can change it by `~/diagnostic_period` parameter), this does not mean that the publish rate is always fixed.